Makefile

```diff
@@ -29,25 +29,28 @@ stop:
 
 # Run tests for the library
 test:
-	poetry export -f requirements.txt --without-hashes --with test --output requirements.txt
-	docker compose -f docker-compose.dev.yml up -d --build --wait
-	docker compose exec -T backend pytest --cov=app
-	docker compose -f docker-compose.dev.yml down
+	- poetry export -f requirements.txt --without-hashes --with test --output requirements.txt
+	- docker compose -f docker-compose.dev.yml up -d --build --wait
+	- docker compose exec -T backend pytest --cov=app
+	- docker compose -f docker-compose.dev.yml down
+
+build-client:
+	pip install -e client/.
 
 # Run tests for the Python client
 test-client:
-	poetry export -f requirements.txt --without-hashes --output requirements.txt
-	docker compose -f docker-compose.dev.yml up -d --build --wait
-	cd client && pytest --cov=pyroclient tests/ && cd ..
-	docker compose -f docker-compose.dev.yml down
+	- poetry export -f requirements.txt --without-hashes --output requirements.txt
+	- docker compose -f docker-compose.dev.yml up -d --build --wait
+	- cd client && pytest --cov=pyroclient tests/ && cd ..
+	- docker compose -f docker-compose.dev.yml down
```

Comment:
Could you add a comment in this file explaining the tab + union please? I didn't know about that, let's try to avoid having someone remove those in the future thinking it was a mistake :)